### DISCUSSION TITLE
Documented behavior for wire:model.change does not work

### DISCUSSION THREAD

**Discussion by lele85** [livewire/livewire#7254] [7. Bugs (archived)] [answered]
### Livewire version

3.0.5

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

This part of the doc the `.change` modifier will send the update on the model on a select when the value changes.

https://livewire.laravel.com/docs/wire-model#updating-on-change-event

This is a simple component:
```php
<?php

namespace App\Livewire;

use Livewire\Component;

class Selector extends Component
{
    public $selected = '1';

    public function render()
    {
        return view('livewire.selector');
    }
}
```

This is the template:
```blade
<div>
    <select wire:model.change="selected">
        <option value="1">One</option>
        <option value="2">Two</option>
        <option value="3">Three</option>
    </select>
    <p>You have selected: {{ $selected }}</p>
</div>
```

I expected the the change to be reflected immediately on the template, instead it never updates.

Switching to:
```blade
<div>
    <select wire:model.live="selected">
        <option value="1">One</option>
        <option value="2">Two</option>
        <option value="3">Three</option>
    </select>
    <p>You have selected: {{ $selected }}</p>
</div>
```

does the trick. In case if is an expected behavior i think the docs should be updated accordingly, if not maybe there's a bug hidden somewhere. :)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PhiloNL** [comment, marked as answer]
Hi @lele85,
Can you try this again with the latest version?
When using v3.1.0 it seems to work as expected. The value is updated as soon as I select a different value:

https://wirebox.app/b/4zno4

**Reply by lele85** [reply]
I confirm that with the latest version the behavior is correct. Thanks!
